paasta_tools/kubernetes_tools.py

```diff
@@ -1937,9 +1934,28 @@ def get_pod_anti_affinity(self) -> Optional[V1PodAntiAffinity]:
                         label_selector=label_selector,
                     )
                 )
+        return affinity_terms
+
+    def get_pod_anti_affinity(self) -> Optional[V1PodAntiAffinity]:
+        """
+        Converts the given anti-affinity on service and instance to pod
+        affinities with the "paasta.yelp.com" prefixed label selector
+        :return:
+        """
+
+        required_terms = self.get_pod_anti_affinity_terms(
+            config_key="anti_affinity",
+        )
+        preferred_terms = self.get_pod_anti_affinity_terms(
+            config_key="anti_affinity_preferred",
+        )
```

Comment:
i think ideally we'd have something that looked like:
```YAML
anti_affinity:
    preferred:
        - ...
    required:
        - ...
```

but we probably don't wanna deal with doing the migration to a new config format right now :p